angular/src/app/service/api/employee/employee.service.ts

```diff
@@ -104,4 +104,8 @@ export class EmployeeService extends BaseApiService {
   public quitJobToOtherTool(input): Observable<ApiResponseDto<any>>{
     return this.processPost(`QuitJobToOtherTool`, input);
   }
+ 
+  public deleteWithEmail(id: number, deteleUserWithEmail: boolean): Observable<any> {
```

Comment:
deleteEmployeeAndDeleteUser(employeeId: number, isDeleteUser: boolean) {
         return this.processDelete(`deleteEmployeeAndDeleteUser?id=${employeeId}&isDeleteUser=${isDeleteUser}`);
}